### PR TITLE
Add token symbol and name fields to NFT rewards

### PR DESCRIPTION
**Pull request by johnnyd-eth**
## What does this PR do and why?

- These fields are key for StudioDAO's projects so would be great to get out ASAP

Screen Shot 2022-08-01 at 3 29 28 pm

- Also extends NFT upload limit 10x since we're not using the cloud function anymore 

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/index.tsx [hunk 1]
```diff
@@ -103,6 +119,39 @@ export default function NftDrawer({
           }}
         >
           <p>{NFT_REWARDS_EXPLAINATION}</p>
+          <Form layout="vertical">
```
Comment (tomquirk) [inline src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/index.tsx:122]: Can add requiredMark="optional" here and remove the optional text in the label

File: src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/index.tsx [hunk 2]
```diff
@@ -103,6 +119,39 @@ export default function NftDrawer({
...
+            <Form.Item
+              label={
+                <TooltipLabel
```
Comment (tomquirk) [inline src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/index.tsx:125]: Can remove the tooltip component and use the `tooltip` prop on form.item instead

**Review by tomquirk — CHANGES_REQUESTED** [review]
@johnnyd-eth LGTM coupla suggestions

**Comment by johnnyd-eth** [comment]
Screen Shot 2022-08-02 at 6 54 28 pm

Screen Shot 2022-08-02 at 6 54 31 pm

This is how it looks after those two comments @aeolianeth. Just me or it look kinda funky?

**Comment by johnnyd-eth** [comment]
I've pushed changes addressing those comments, have a play and LMKWYT

**Comment by tomquirk** [comment]
@johnnyd-eth perfect thanks. added a sneaky empty

Screen Shot 2022-08-03 at 12 04 16 PM

**Review by tomquirk — APPROVED** [review]

**Pull request merged**